common/changes/@azure/identity-cache-persistence/user-altinokd-bump-msal-node-ext_2023-12-01-20-48.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
What is this file for?  This seems to be a mistake to check this in

Reply:
yes, please remove

Reply:
Removed.